os-autoinst-obs-auto-submit

```diff
@@ -232,6 +231,7 @@ is_staging_specified() {
 
 submit() {
     # submit from staging project if specified
+    is_staging_specified && src_project=$staging_project must_cleanup_staging=1
```

Comment:
Can we just inline the condition as the function is now used only here?

Reply:
I guess we can.. but from my POV is a little better than have a long line. I think it is more readable with the function

Reply:
So that would be

```suggestion
    [[ -n $staging_project && $staging_project != none && $staging_project != "$src_project" ]] || src_project=$staging_project must_cleanup_staging=1
```

right?

Reply:
yea, this is the suggestion. I am just saying that even if the function is ony used here, it is a little better to read it using the function

Reply:
I would at least keep the `if` and assignments in separate lines

```suggestion
    if [[ -n $staging_project && $staging_project != none && $staging_project != "$src_project" ]]; then
        src_project=$staging_project
        must_cleanup_staging=1
    fi
```

Reply:
I prefer the inlined same as Jan

Reply:
In this case, you want to override the `src_project` if staging_project is set (and clean it if different from `src_project`), so `&&` is correct.

Reply:
My point is the if is more legible 🙃